README.md

```diff
@@ -75,12 +75,14 @@ components = {
 }
 ```
 
-At a minimum, you will be required to declare the bundle's required number of cpus, its memory and diskspace for your project's build file. For a typical microservice this may look like the following:
+At a minimum, you will be required to declare the bundle's required number of cpus, its memory and disk space for your project's build file. For a typical microservice this may look like the following:
 
 ```scala
+import ByteConversions._
+
 BundleKeys.nrOfCpus := 1.0
-BundleKeys.memory := "64m"
-BundleKeys.diskSpace := "10m"
+BundleKeys.memory := 64.MiB
```

Comment:
Yes of course, for normal code, but DSLs are different, many things are allowed there.